security/crowdsec/src/etc/cron.d/oscrowdsec.cron

```diff
@@ -0,0 +1,9 @@
+# DO NOT EDIT THIS FILE -- OPNsense auto-generated file
+#
+# User-defined crontab files can be loaded via /etc/cron.d
+# or /usr/local/etc/cron.d and follow the same format as
+# /etc/crontab, see the crontab(5) manual page.
+SHELL=/bin/sh
+PATH=/etc:/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin:/usr/local/sbin
+#minute	hour	mday	month	wday	who	command
+0	1,13	*	*	*	root	/usr/local/opnsense/scripts/OPNsense/CrowdSec/hub-upgrade.sh
```

Comment:
note: there are 2 types of standard cron-inclusion which this one isn't but this certainly also works